tests/integration/components/user-editor/component-test.js

```diff
@@ -0,0 +1,18 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
Oh yes! Particularly with unit tests and component integration tests - not got much experience with acceptance tests so far though. Having been bitten by regressions in the past I'm definitely a fan of well-tested code... 😉